inst/rmarkdown/templates/xaringan/resources/su.css

```diff
@@ -0,0 +1,135 @@
+.remark-slide-content {
+  border-top: 5px solid #002F5F;
+  border-bottom: 5px solid #002F5F;
+  font-size: 1.2em;
+  padding: 20px 60px;
+}
+
+.remark-slide-content > h1 {
+  margin-top: 20px;
+  margin-bottom: 0px;
+}
+
+.inverse {
+  background-color: #002F5F;
+  border:none;
+  font-color: #fff;
+  text-shadow: none;
+}
+
+.inverse, .inverse h1, .inverse h2, .inverse h3, .inverse h4 {
+  color: #fff;
+}
+
+.inverse code {
+  color: #EC008C;
+}
+
+.footnote {
+  border-top: 1px solid #002F5F;
+  font-size: 0.6em;
+}
+
+.remark-code{
+  color: #500000;
+}
+
+.remark-inline-code {
+  color: #500000;
+}
+
+.inverse .remark-code {
+  background-color: #ebf0f5;
+}
+
+.inverse .remark-inline-code {
+  background-color: #ebf0f5;
+}
+
+.remark-code-line-highlighted {
+  background-color: rgba(225, 224, 0, 0.6);
+}
+
+.inverse .remark-code-line-highlighted {
+  background-color: #f7dfcc;
+}
+
+.inverse .hljs-github {
+    background: #33597f;
+}
+
+.inverse .hljs {
+    background: #33597f;
+}
+
+h1, h2, h3, h4 {
+  color: #002F5F;
+}
+
+code {
+  color: #002F5F;
+}
+
+a {
+  color: #3F97B5;
+  text-decoration: none;
+  transition: all .2s ease-in-out;
+}
+
+a:hover {
+    font-size: 1.2em;
+    color: #d95e00;
+}
+
+.inverse a {
+  color: #d95e00;
+  text-decoration: none;
+  transition: all .2s ease-in-out;
+}
+
+.inverse a:hover {
+    font-size: 1.2em;
+    color: #a3a86b;
+}
+
+.remark-slide-number {
+  font-size: 0.6em;
+}
+
+.title-slide {
+  background-image: url(https://pp-prod-admin.it.su.se/polopoly_fs/1.403917.1538464558!/menu/standard/file/SU_logo_NEG_ENGELSK.gif);
```

Comment:
I did indeed base the theme on `uo.css`; how should I properly thank @datalorax? As a comment in `su.css`?

Regarding the logo, here I have run into some red tape: apparently (https://goo.gl/VBoq4w; https://goo.gl/arLEC5 in Swedish with more detail), only staff of the university are allowed to use the logo in print and presentation material, while others have to seek explicit permission. I will see if I can resolve this somehow; otherwise I'll have to withdraw my pull request.

Reply:
yes, something like that as a comment in css would be great. We always give credit to others' work in an open source community.

as for the school guidelines, you may put the logo down or ask your school secretary for help(usually it's not that serious, but i'm not sure how serious you uni is on this issue.)

Reply:
I think it suffices to acknowledge @datalorax in `NEWS.md`.

For the logo, that is really sad. If the copyright issue can't be resolved, I don't know if we could merge this PR. Anyway, since this theme is based on another existing theme, are you able to figure out the _minimal_ CSS required for your theme? I mean you could provide some minimal CSS to override `uo.css`: https://bookdown.org/yihui/rmarkdown/css-and-themes.html

```yaml
output:
  xaringan::moon_reder:
    css: ['default', 'uo', 'uo-fonts', 'your-minimal-su.css']
```

Reply:
Thanks everybody. I appreciate the acknowledgement. I'm totally comfortable with whatever you all decide is best. And I'm happy to see the `uo.css` code being helpful to others!

Reply:
As suggested I'd also prefer the rebase on the already existing theme.

Did you resolve the copyright questions meanwhile?
Also having a link to a URL from a school is most likely to break in the future.